src/app/home/home.component.html

```diff
@@ -1 +1,51 @@
-<h2>Home - Team 22</h2>
\ No newline at end of file
+<div class="home-container">
+    <div class="hero">
+      <img src="images/logo.png" alt="ColorCoord Logo" class="hero-logo">
+      <h1>Welcome to ColorCoord</h1>
+      <p class="tagline">The ultimate tool for creating color coordinate sheets</p>
+      
+      <div class="cta-container">
+        <a routerLink="/drawing" class="cta-button">Create Your Color Coordinate Sheet</a>
+      </div>
+    </div>
+    
+    <div class="about-section">
+      <h2>About ColorCoord</h2>
+      <p class="about-description">
+        ColorCoord is a cutting-edge web application designed for artists, educators, and designers 
+        who need to create precise color coordinate sheets. Our easy-to-use platform allows you to 
+        generate custom coordinate grids with precise color mapping.
+      </p>
+      
+      <div class="features">
+        <div class="feature">
+          <div class="feature-icon">📊</div>
+          <h3>Customizable Grids</h3>
+          <p>Create grids with up to 1000 rows and 702 columns to fit your specific needs.</p>
+        </div>
+        
+        <div class="feature">
+          <div class="feature-icon">🎨</div>
+          <h3>Color Selection</h3>
+          <p>Choose from 10 vibrant colors to bring your coordinate sheets to life.</p>
+        </div>
+        
+        <div class="feature">
+          <div class="feature-icon">🖨️</div>
+          <h3>Print Ready</h3>
+          <p>Print your creations with a single click for use in classrooms or design projects.</p>
+        </div>
+      </div>
+      
+      <div class="get-started">
+        <h2>Ready to get started?</h2>
+        <p>Click below to begin creating your first color coordinate sheet.</p>
+        <a routerLink="/drawing" class="secondary-cta-button">Start Creating Now</a>
+      </div>
+    </div>
+  </div>
+
+
+<footer>
```

Comment:
There is a double footer, one in the routing component, one in the app. I think this one needs to be removed.